### PR TITLE
The first 3 sections of the ruby course (intro, numbers and bools)

### PR DESCRIPTION
**Pull request by roa-sh**
Hi esteban, this is my first PR.
I created another branch called my_work, here i'm uploading my files, and then i'll ask you for a PR for merging with main branch.

Thanks!

### CODE REVIEW

**Comment by roa-sh** [comment]
Hi esteban, i continued working and added the section 4,5, and 6.

File: 2section_getting_started.rb [hunk 1]
```diff
@@ -0,0 +1,85 @@
+=begin
+    This code was made for practicing ruby :D
+    Author: @roa-sh
+=end
+
+5.times{puts "Hello world for everybody"}
+puts
+
+#-----------------The puts method---------
+#and can prints aritmetic operations too
```
Comment (estebancordoba) [inline 2section_getting_started.rb]: for comments put a space after the # (https://github.com/rubocop/ruby-style-guide#hash-space)

File: 2section_getting_started.rb [hunk 2]
```diff
@@ -0,0 +1,85 @@
+=begin
+    This code was made for practicing ruby :D
+    Author: @roa-sh
+=end
+
+5.times{puts "Hello world for everybody"}
+puts
+
+#-----------------The puts method---------
```
Comment (estebancordoba) [inline 2section_getting_started.rb]: Avoid hyphenated comments

File: 5section_methods_plus_calc.rb [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        return n1*n2
+    when 4 
+        return n1.to_f/n2
+    else
+        return "Please enter a valid operation"
+    end
+end
+
+def are_odd(n1, n2)
+    # p "in odd"
```
Comment (estebancordoba) [inline 5section_methods_plus_calc.rb]: For the indentation use 2 spaces: https://github.com/rubocop/ruby-style-guide#spaces-indentation

File: 6section_methods_conditionals2.rb [hunk 4]
```diff
@@ -0,0 +1,33 @@
+def look_for_place(credentials, age, nat)
+    p nat.upcase == "JAPANESE"
+    if nat.upcase == "JAPANESE"
+        case credentials
```
Comment (estebancordoba) [inline 6section_methods_conditionals2.rb]: Fix the indentation please

**Review by estebancordoba — CHANGES_REQUESTED** [review]
I recommend you read the ruby style guide: https://github.com/rubocop/ruby-style-guide, it is also part of the path

**Comment by roa-sh** [comment]
Ok esteban, i'll review the guide and apply changes.

**Comment by roa-sh** [comment]
Hi, @estebancordoba i fixed the things you commented me. Thnaks for the rubocop recomendation, i started using it as my linter and also helped me to the good practices in coding.

**Review by estebancordoba — APPROVED** [review]

**Pull request merged**